osx/make_app.sh

```diff
@@ -0,0 +1,119 @@
+"
+$full_target_python $downloads/get-pip.py -f $wheels --prefix=$local_sys_prefix
+
+# Install Plover
+$full_target_python -m utils.install_wheels --ignore-installed --prefix=$local_sys_prefix $plover_wheel
+
+# Make launcher
+plover_executable=MacOS/Plover
+launcher_file=$app_dir/Contents/$plover_executable
+cp $osx_dir/app_resources/plover_launcher.sh $launcher_file
+sed -i '' "s/pythonexecutable/$target_python/g" $launcher_file
+chmod +x $launcher_file
+
+# Copy icon
+cp $osx_dir/plover.icns $app_dir/Contents/Resources/plover.icns
+
+# Get Plover's version
+plover_version=$($full_target_python -c "from plover import __version__; print(__version__)")
+
+# This allows notifications from our Python binary to identify as from Plover...
+/usr/libexec/PlistBuddy -c 'Add :CFBundleIdentifier string "org.openstenoproject.plover"' $app_dir/Contents/Frameworks/Info.plist
+
+# Setup PList for Plover
+/usr/libexec/PlistBuddy -c 'Add :CFBundleDevelopmentRegion string "en"' $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c 'Add :CFBundleIconFile string "plover.icns"' $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c 'Add :CFBundleIdentifier string "org.openstenoproject.plover"' $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c 'Add :CFBundleName string "Plover"' $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c 'Add :CFBundleDisplayName string "Plover"' $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c "Add :CFBundleExecutable string \"$plover_executable\"" $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c 'Add :CFBundlePackageType string "APPL"' $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c "Add :CFBundleShortVersionString string \"$plover_version\"" $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c "Add :CFBundleVersion string \"$plover_version\"" $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c 'Add :CFBundleInfoDictionaryVersion string "6.0"' $app_dir/Contents/Info.plist
+year=$(date '+%Y')
+copyright="© $year, Open Steno Project"
+/usr/libexec/PlistBuddy -c "Add :NSHumanReadableCopyright string \"$copyright\"" $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c 'Add :NSPrincipalClass string "NSApplication"' $app_dir/Contents/Info.plist
+/usr/libexec/PlistBuddy -c 'Add :NSAppSleepDisabled bool true' $app_dir/Contents/Info.plist
+
+# Remove broken symlinks
+rm $target_dir/Python.framework/Headers
+rm $target_dir/Python.framework/Python
+rm $target_dir/Python.framework/Resources
+# Trim superfluous content from app
+sed -e "s/\$python_version/$py_version/" -e "s/\$target_python/$target_python/" osx/app_resources/dist_blacklist.txt > build/dist_blacklist.txt
+$python -m utils.trim $target_dir/Python.framework build/dist_blacklist.txt
+# Make distribution source-less
+$python -m utils.source_less $target_libs */pip/_vendor/distlib/*
+
+# Strip 32-bit support
+ditto -v --arch x86_64 $app_dir $stripped_dir
+rm -rf $app_dir
+rm -rf $app_dist_dir
```

Comment:
This should be done at the beginning, so only the results of the current script run are left (if the script failed, there should be no `$app_dist_dir`).